edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Amqp/linkhandlers/ModuleMessageLinkHandler.cs

```diff
@@ -45,13 +46,17 @@ static class Metrics
             static readonly IMetricsCounter MessagesMeter = Util.Metrics.Metrics.Instance.CreateCounter(
                 "messages_sent",
                 "Messages sent to module",
-                new List<string> { "protocol", "from", "to" });
+                new List<string> { "from", "to", "from_route_output", "to_route_input" });
 
             public static void AddMessage(IIdentity identity, IMessage message)
             {
                 string from = message.GetSenderId();
                 string to = identity.Id;
-                MessagesMeter.Increment(1, new[] { "amqp", from, to });
+
+                // Note these are switched, because the message's output is edgeHub's input and vice-versa
```

Comment:
Emm.. this does not make sense. EdgeHub has no concepts of Input/outputs. Those are the module concepts.

Reply:
I meant it in terms of 'from' being edgeHub's input (the message's output tag) and 'to' being edgeHub's output (the message's input tag).

Reply:
actually, looking at the actual prometheous tag it corresponds to, I'll rename my variables.

Reply:
Its now fromRouteOutput and toRouteInput